### PR TITLE
Change suite number and PO box number to Strings

### PR DESCRIPTION
**Pull request by jlorentson**

### CODE REVIEW

**Comment by ctmay4** [comment]
The `intValue` and `doubleValue` methods should still be dealt with. 

The `intValue` method is no longer needed I believe.  It should be removed.

The `doubleValue` method is still used for lon/lat.  It's probably safe to assume they are always numeric, but maybe that method should be safer?

**Comment by jlorentson** [comment]
@ctmay4 Makes sense. Return null if not numeric?
intValue() is still used for status code and feature match count, both of which make sense to me to leave as Integers.

**Comment by ctmay4** [comment]
I didn't realize it was used.

I guess for now we can just return null.  You can catch a NumberFormatException and return null for those two methods.  Please also add unit tests.

**Comment by criderp** [comment]
I agree, it makes sense to leave those 2 as integers.

**Comment by ctmay4** [comment]
The build failed.

File: src/test/java/com/imsweb/geocoder/GeocodeOutputTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+public class GeocodeOutputTest {
+
+    // The important behavior in the GeocodeOutput class is tested in GeocoderTest, but wanted a test for some of the helper methods
+    @Test
+    public void testIntValue() throws Throwable {
+        Class<?>[] clazz = {String.class};
+
+        Object[] param = {"7"};
+        Object result = PrivateAccessor.invoke(GeocodeOutput.class, "intValue", clazz, param);
```
Comment (ctmay4) [inline src/test/java/com/imsweb/geocoder/GeocodeOutputTest.java:19]: As an FYI for the future, I've never been a fan of testing private methods in this way.  I think its good you tested the method, put I've always found the path of least resistance is to make the method package private.  We can leave this for now.

Comment (jlorentson) [reply src/test/java/com/imsweb/geocoder/GeocodeOutputTest.java:19]: I agree, actually, about using the PrivateAccessor, but I was reluctant to change access modifier. I'll be less reluctant in the future!

**Pull request merged**